lib/config.js

```diff
@@ -17,7 +17,7 @@ config.postgresqlUser = 'postgres';
 config.postgresqlPassword = null;
 config.postgresqlDatabase = 'postgres';
 config.postgresqlHost = 'localhost';
-config.postgresqlPoolSize = 100;
+config.postgresqlPoolSize = 50;
```

Comment:
My local Postgres was erroring with something like "sorry, there are already too many clients" when I saturated PL with requests. This fixed it. Idk if we actually want to change the default though? This mostly got committed on accident 😅